entity-service-change-event-api/build.gradle.kts

```diff
@@ -0,0 +1,68 @@
+import com.google.protobuf.gradle.generateProtoTasks
+import com.google.protobuf.gradle.id
+import com.google.protobuf.gradle.ofSourceSet
+import com.google.protobuf.gradle.plugins
+import com.google.protobuf.gradle.protobuf
+import com.google.protobuf.gradle.protoc
+
+plugins {
+  `java-library`
+  id("com.google.protobuf") version "0.8.15"
+  id("org.hypertrace.publish-plugin")
+}
+
+val generateLocalGoGrpcFiles = false
```

Comment:
we don't need either the go or grpc targets here (if and when we need go, we actually handle that differently now anyway). A lot falls out - I think it resolves down to something like below
```kts

plugins {
  `java-library`
  id("com.google.protobuf") version "0.8.17"
  id("org.hypertrace.publish-plugin")
}

protobuf {
  protoc {
    artifact = "com.google.protobuf:protoc:3.18.0"
  }
}

dependencies {
  api("com.google.protobuf:protobuf-java:3.18.0")
}

sourceSets {
  main {
    java {
      srcDirs("build/generated/source/proto/main/java")
    }
  }
}
```

Reply:
done